v1/lib/server/readMetadata.js

```diff
@@ -54,6 +57,14 @@ function getDocsPath() {
   return siteConfig.customDocsPath ? siteConfig.customDocsPath : 'docs';
 }
 
+function shouldGenerateNextReleaseDocs() {
+  return !(
+    env.versioning.enabled &&
+    program.name() === 'docusaurus-build' &&
```

Comment:
Moving this to `generate.js` essentially remove this `program.name() === 'docusaurus-build`